jedi/inference/compiled/subprocess/__init__.py

```diff
@@ -156,11 +156,21 @@ class CompiledSubprocess(object):
     # Start with 2, gets set after _get_info.
     _pickle_protocol = 2
 
-    def __init__(self, executable):
+    def __init__(self, executable, env_vars={}):
         self._executable = executable
+        self._env_vars = env_vars
```

Comment:
Done. Thank you for the feedback. I went ahead and added kwargs for `find_system_environments` and `get_system_environments` so that the possibility is there without breaking the API.